tests/scripts/minikube.sh

```diff
@@ -109,6 +109,7 @@ case "${1:-}" in
       wait_for_ssh
     fi
     # create a link so the default dataDirHostPath will work for this environment
+    minikube ssh "sudo mkdir -p /mnt/sda1/${PWD}; sudo mkdir -p $(dirname $PWD); sudo ln -s /mnt/sda1/${PWD} $(dirname $PWD)/"    
```

Comment:
what directory will this create? does it need a new comment?

Reply:
added